custom_components/lennoxs30/climate.py

```diff
@@ -619,3 +608,12 @@ async def async_set_fan_mode(self, fan_mode):
                 _LOGGER.error("climate:async_set_fan_mode - error:" + e.message)
             else:
                 _LOGGER.error("climate:async_set_fan_mode - error:" + str(e))
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return device info."""
+        return {
+            "name":  self._system.name,
+            "identifiers": {(DOMAIN, self._system.unique_id())},
+            "manufacturer": "Lennox",
+            "model": "Lennox S30",
```

Comment:
I think there are also E30s which use same protocol. 

 Should be.  “Lennox “ + _system.productType